mm2src/coins/utxo.rs

```diff
@@ -560,10 +581,26 @@ pub trait UtxoCommonOps {
 
 #[async_trait]
 pub trait UtxoStandardOps {
-    /// Gets tx details by hash requesting the coin RPC if required
-    async fn tx_details_by_hash(&self, hash: &[u8]) -> Result<TransactionDetails, String>;
+    /// Gets tx details by hash requesting the coin RPC if required.
+    /// If you plan to call this method for multiple transactions (for example, when loading transaction history),
+    /// It's recommended to use this method for transactions ordered by **ascending** height for the best performance.
```

Comment:
This comment seems confusing - it states that transactions should be ordered by height that can't be applied to the input HashMap.

Reply:
> I put this comment before realizing that the history transactions are requested in descending order of height

The main confusing point is not how transactions requested in history - the input of this method is HashMap that can not be ordered at all.

Reply:
Done

Reply:
I see, then it's worth replacing the comment with this description as it seems more clear :slightly_smiling_face:

Reply:
Unfortunately, we request the most recent transaction first, so this comment doesn't mean anything anyway :D

Reply:
I see, thanks for clarification :slightly_smiling_face: